manifests/cf-manifest/cloud-config/000-base-cloud-config.yml

```diff
@@ -0,0 +1,21 @@
+---
+azs:
+- name: z1
+  cloud_properties: {availability_zone: eu-west-1a}
+- name: z2
+  cloud_properties: {availability_zone: eu-west-1b}
+- name: z3
+  cloud_properties: {availability_zone: eu-west-1c}
+
+compilation:
+  vm_type: compilation
+  az: z1
```

Comment:
So, you can only compile in one AZ? Well, I guess it is not a huge risk, one can always change that.